Sources/BraintreePayPal/BTPayPalReturnURL.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+enum BTPayPalAppSwitchReturnURLState {
```

Comment:
I see that `BTPayPalAppSwitchReturnURLState` wasn't renamed to `BTPayPalReturnURLState`. Did you mean to leave it the same?

Reply:
✅ : https://github.com/braintree/braintree_ios/pull/1271/commits/58403e1c50e1cd7ad2cea5deae6d46a6a3c898fe